<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMenu</class>
 <widget class="QMainWindow" name="UserMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Меню пользователя</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>405</width>
      <height>279</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlMenu">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QLabel" name="welcome">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Добро пожаловать, [username]!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMyBooks">
       <property name="text">
        <string>Мои книги</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOffers">
       <property name="text">
        <string>Объявления</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSwaps">
       <property name="text">
        <string>Обмены</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMyReviews">
       <property name="text">
        <string>Мои отзывы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
